CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### Deprecations/Changes
```

Comment:
Since this is mostly used by devs, I think this should not be mentioned in changelog, but happy to discuss.

Reply:
Agreed but I wanted to include it just in case someone still wanted it include.

Reply:
Removing changelog edits.